hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -319,6 +331,25 @@ public void setBlockDeletionLimit(int limit) {
     this.blockLimitPerInterval = limit;
   }
 
+  @Config(key = "hdds.datanode.volume.min.free.space",
```

Comment:
DatanodeConfiguration has the key prefix "hdds.datanode" defined.

Reply:
This is now valid after HDDS-12424.  It makes searching for config key easier.